otter/test/convergence/test_service.py

```diff
@@ -330,13 +359,13 @@ def test_unexpected_errors(self):
         dirty flag.
         """
         expected_error = RuntimeError('oh no!')
-        sequence = SequenceDispatcher([
+        sequence = [
```

Comment:
Oh, that's a really good point. I'd really rather avoid using the reference_dispatcher at all in our tests, and make sure everything is explicitly tracked, but I didn't want to go and update all of them now. I will make the change for these failure tests as you suggest :)